### DISCUSSION TITLE
How to modify the color of .navbar-dark?

### DISCUSSION THREAD

**Discussion by Elindo586** [twbs/bootstrap#36371] [Q&A] [answered]
I have this piece of code here:

```
<nav
      className="navbar navbar-expand-lg bg-dark navbar-dark"
      onClick={title}
    >
      <Link href="/" className="navbar-brand" onClick={title}>
        <a>TECHNICAL UNION</a>
      </Link>
```
in my CSS I have this like this:

```
.navbar-dark {
  background: #016698 !important;
}
```

The color should go from dark to a blueish color.

This code has worked in REACT, but I am moving the site to Next.js and it isn't working... I also have the newest bootstrap....   has anything changed?

How do I change the navbar to the color I want?

**Comment by coliff** [comment]
You don’t need the `bg-dark` class on your `nav`

**Comment by mdo** [comment, marked as answer]
https://getbootstrap.com/docs/5.2/components/navbar/#color-schemes might provide some more answers here. `.navbar-dark` changes the text and link colors in the navbar, but not the `background-color`. Your problem is that `.bg-dark` is a utility class with `!important` and that's likely overriding your intended changes. Drop the `.bg-dark` and your custom CSS should work.

**Comment by ghost** [comment]
You dont need bg-dark in the navbar, instead, you could use custom and use !important
